code/modules/clothing/spacesuits/chronosuit.dm

```diff
@@ -33,7 +33,7 @@
 	var/datum/action/innate/chrono_teleport/teleport_now = new
 	var/activating = 0
 	var/activated = 0
-	var/cooldowntime = 50 //deciseconds
+	var/cooldowntime = 20 //deciseconds
```

Comment:
This has nothing to do with teleporting, it's how often you can activate and deactivate the suit.